.gitignore

```diff
@@ -80,3 +81,9 @@ phpstan*.neon
 # dev scripts loaded via composer
 /shell/update-copyright.php
 /shell/translations.php
+
+# test files
+**test*.php
```

Comment:
Sorry for my poor knowledge, but what does it match?

Reply:
if it only should match files prefixed with test, then I think there is a / missing and it should be `**/test*.php`

Reply:
@kiatng no default files, but in nearly every project hast some test files .;)

Reply:
It's true, I have them, just not name with the word "test'.